app/views/welcome/index.html.erb

```diff
@@ -4,9 +4,9 @@
       <h1 class="jumbotron-heading">Find OpenSource Mentor</h1>
       <p class="lead text-muted">A platform to connect junior developers with open source projects</p>
       <div class="row d-flex justify-content-center">
-        <%= button_to 'Sign up as junior', user_github_omniauth_authorize_path, method: :get, class: "btn btn-primary my-2 margin-btn" %>
-        <%= button_to 'Sign up as mentor', user_github_omniauth_authorize_path, method: :get, class: "btn btn-secondary my-2 margin-btn" %>
+        <%= button_to 'Sign up as mentee', user_github_omniauth_authorize_path, method: :get, params: { role: 'mentee' }, class: "btn btn-primary my-2 margin-btn" %>
```

Comment:
This implementation allows "hackers" to prepare requests with whichever role they like. Which has no implications in the current shape, but if we decide to add "admin" role, then this might be insecure.

For now, it's fine.

Reply:
Yes, you are right, from the next PR will be more careful)